python/generate-short-url/README.md

```diff
@@ -0,0 +1,61 @@
+# 📱 Validate phone number and get it's country information
+
+A Python Cloud Function that generates short URLs with help of Bitly or Tinyurl.
+
+_Example input:_
+
+```json
+{
+	"provider": "tinyurl", 
+	"url": "https://github.com/open-runtimes/examples",
+	"api_key": "API_KEY"
+}
+```
+
+
+_Example output:_
```

Comment:
Let's add one more output example. THe one we return when something fails (success=false, and a message).